services/core/shared/core/transactions.js

```diff
@@ -34,11 +34,20 @@ const mergeTransactions = async (params) => {
 		meta: {},
 	};
 
+	const transactions = {
+		data: [],
+		meta: {},
+	};
+
 	const { offset, limit, ...remParams } = params;
 	params = remParams;
 
 	const pendingTxs = await getPendingTransactions(params);
-	const transactions = await coreApi.getTransactions(params);
+	const response = !params.id ? await coreApi.getTransactions(params) : {};
```

Comment:
Could you please elaborate on why do we need this condition check here?

Reply:
Clarified offline!